shared/locales/de/website-partners.json

```diff
@@ -283,6 +268,7 @@
 		"org-photo": ""
 	},
 	"sierra-leone-association-of-ebola-survivors": {
+		"firestore-id": "slaes",
```

Comment:
A key that is the same in every translation file, seems wrong. This file should be for strings that differ [across](url) languages.